spk/sslh/src/start-stop-status

```diff
@@ -46,17 +37,12 @@ stop_daemon ()
         echo `date`" : Stop SSLH..." >> $SSLHLOG
         kill $PID
     fi
-	
-    # Remove the application from the main interface if it was previously added.
-    if [ -h $SYNO3APP/sslh ]
-    then
-		rm -f $SYNO3APP/sslh
-    fi
 }
 
 daemon_status ()
 {
-	[ -h $SYNO3APP/sslh ]
+    PID=`ps | grep $SSLH | grep -v grep |awk '{print $1}'`
```

Comment:
Couldn't you assign a PID file with startup option `-P`, and get the status from that?